### PR TITLE
Adapt to https://github.com/coq/coq/pull/19310

### PR DESCRIPTION
**Pull request by proux01**
Adapt to https://github.com/coq/coq/pull/19310

This should be backward compatible and could be merged without waiting

### CODE REVIEW

**Comment by gares** [comment]
This looks like a mess. @FissoreD would it be possible to support both `Coq` and `Stdlib` as the root *and* omit that component from the class name?

File: apps/tc/elpi/tc_aux.elpi [hunk 1]
```diff
@@ -93,7 +93,8 @@ namespace tc {
     if (tc.is-option-active tc.oTC-clauseNameShortName)
       (Path = "")
       (coq.gref->path Gr [Hd | Tl], 
-        std.fold Tl Hd (x\acc\r\ r is acc ^ "." ^ x) Path', 
+        if (Hd = "Coq") (Hd' = "Stdlib") (Hd' = Hd),
+        std.string.concat "." [Hd'|Tl] Path',
```
Comment (gares) [inline apps/tc/elpi/tc_aux.elpi:97]: I pushed a nit, tested locally

**Comment by proux01** [comment]
@gares no hurry but my understanding is that this can be merged

**Comment by gares** [comment]
sure

**Pull request merged**